contribs/gmf/src/datasource/datasourcebeingfiltered.js

```diff
@@ -1,23 +0,0 @@
-goog.provide('gmf.datasource.DataSourceBeingFiltered');
```

Comment:
The renaming for this file got lost. 
contribs/gmf/src/datasource/DatasourceBeingFiltered.js

Reply:
It's because it's nearly empty and I've changed everything... I think I can't proceed differently.

Reply:
To make it work, you would have to split the move and the change of content into two commits. It is annoying...

Reply:
Done, but I'm not sure if that have changed something (except that I have good clear conscience).